src/app.config.ts

```diff
@@ -4,6 +4,12 @@ export abstract class AbstractAppConfig {
   abstract getApiUrl(): string;
   abstract getCaseDataUrl(): string;
   abstract getDocumentManagementUrl(): string;
+  getDocumentManagementUrlV2(): string {
+    return undefined;
```

Comment:
what's the point return the undefined

Reply:
That sets the default value, hence making the change a non-breaking change